enchant/tests.py

```diff
@@ -573,24 +575,34 @@ def test_py2exe(self):
         except ImportError:
             return
         os.environ["PYTHONPATH"] = dirname(dirname(__file__))
-        setup_py = path.join(dirname(__file__),"..","tools","setup.py2exe.py")
+        setup_py = path.join(dirname(__file__), "..", "tools", "setup.py2exe.py")
         if not path.exists(setup_py):
             return
         buildCmd = '%s %s -q py2exe --dist-dir="%s"'
-        buildCmd = buildCmd % (sys.executable,setup_py,self._tempDir)
+        buildCmd = buildCmd % (sys.executable, setup_py, self._tempDir)
         res = runcmd(buildCmd)
-        self.assertEqual(res,0)
+        self.assertEqual(res, 0)
         testCmd = self._tempDir + "\\test_pyenchant.exe"
         self.assertTrue(os.path.exists(testCmd))
         res = runcmd(testCmd)
-        self.assertEqual(res,0)
-    test_py2exe._DOC_ERRORS = ["py","exe"]
-        
+        self.assertEqual(res, 0)
+
+    test_py2exe._DOC_ERRORS = ["py", "exe"]
+
     def _mkdtemp(self):
         import tempfile
         return tempfile.mkdtemp()
 
 
+class TestPy3Import(unittest.TestCase):
+    def test_py3_import(self):
+        if sys.platform == "darwin" and sys.version_info[0] >= 3:
+            import importlib
+            import enchant
+            enchant_finder = importlib.find_loader('enchant') is not None
+            self.assertTrue(enchant_finder)
```

Comment:
Thanks, this is a good test, perhaps we should broaden it rather than only tying it to OSX, with something like:

```
try:
  import importlib
except ImportError:
  pass
else:
  ...the rest of the test...
```

What do you think?